examples/rest-xml-json/README_es.adoc

```diff
@@ -0,0 +1,404 @@
+
+La idea es primero activar los servicios jaxrs. Esto se hace usando
+Anotación @EnableServices.
+
+Luego creamos sobre la marcha la aplicación simplemente devolviendo un objeto
+que representa el web.xml. Aquí simplemente lo usamos para definir el contexto
+root, pero también puede usarlo para definir su aplicación REST. Y para
+completa la definición de la aplicación agregamos la anotación @Classes para definir
+el conjunto de clases que se utilizarán en esta aplicación.
+
+Finalmente para probarlo usamos la API del cliente cxf para llamar al servicio REST en
+métodos get() y post().
+
+Nota al margen: para mostrar que usamos JSON o XML según el método de prueba que
+activado en EnableServices el atributo httpDebug que imprime el
+mensajes http en los registros.
+
+[source,java]
+----
+package org.superbiz.rest;
+
+import org.apache.cxf.jaxrs.client.WebClient;
+import org.apache.openejb.jee.WebApp;
+import org.apache.openejb.junit.ApplicationComposer;
+import org.apache.openejb.junit.Classes;
+import org.apache.openejb.testing.EnableServices;
+import org.apache.openejb.testing.Module;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.ws.rs.core.MediaType;
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+
+@EnableServices(value = "jaxrs", httpDebug = true)
+@RunWith(ApplicationComposer.class)
+public class GreetingServiceTest {
+    @Module
+    @Classes(GreetingService.class)
+    public WebApp app() {
+        return new WebApp().contextRoot("test");
+    }
+
+    @Test
+    public void getXml() throws IOException {
+        final String message = WebClient.create("http://localhost:4204").path("/test/greeting/")
+                .accept(MediaType.APPLICATION_XML_TYPE)
+                .get(Response.class).getValue();
+        assertEquals("Hi REST!", message);
+    }
+
+    @Test
+    public void postXml() throws IOException {
+        final String message = WebClient.create("http://localhost:4204").path("/test/greeting/")
+                .accept(MediaType.APPLICATION_XML_TYPE)
+                .post(new Request("Hi REST!"), Response.class).getValue();
+        assertEquals("hi rest!", message);
+    }
+
+    @Test
+    public void getJson() throws IOException {
+        final String message = WebClient.create("http://localhost:4204").path("/test/greeting/")
+                .accept(MediaType.APPLICATION_JSON_TYPE)
+                .get(Response.class).getValue();
+        assertEquals("Hi REST!", message);
+    }
+
+    @Test
+    public void postJson() throws IOException {
+        final String message = WebClient.create("http://localhost:4204").path("/test/greeting/")
+                .accept(MediaType.APPLICATION_JSON_TYPE)
+                .post(new Request("Hi REST!"), Response.class).getValue();
+        assertEquals("hi rest!", message);
+    }
+}
+----
+
+#Running
+
+Ejecutar el ejemplo es bastante simple. En el directorio ``rest-xml-json''
```

Comment:
Initial quotes are not the same as final single quotes

Reply:
The correction it is maked.